qt/place_page_dialog.cpp

```diff
@@ -84,6 +85,34 @@ PlacePageDialog::PlacePageDialog(QWidget * parent, place_page::Info const & info
     addEntry(DebugPrint(PropID::FMD_CUISINE), cuisines);
 
   QDialogButtonBox * dbb = new QDialogButtonBox();
+
+  QPushButton * toButton = new QPushButton("To");
+  toButton -> setIcon(QIcon(":/navig64/point-finish.png"));
+  connect(toButton, &QAbstractButton::clicked, this, [this]
+  {
+    SetRoutePointAddMode(RouteMarkType::Finish);
+    OnClose();
+  });
+  dbb->addButton(toButton, QDialogButtonBox::ActionRole);
+
+  QPushButton * stopButton = new QPushButton("Stop");
+  stopButton -> setIcon(QIcon(":/navig64/point-intermediate.png"));
+  connect(stopButton, &QAbstractButton::clicked, this,
+          [this]() {
+            this->SetRoutePointAddMode(RouteMarkType::Intermediate);
+            this->OnClose();
+            });
```

Comment:
Formatting here and below is not the same as above.

Reply:
Done